### ISSUE TITLE
Support for angular 2 above 2.1.1

### ISSUE THREAD

**Issue by phil-cap** [akveo/ng2-smart-table#81]
Any plans to support 2.1.2 and upcoming versions? The peer dependency is currently fixed at 2.1.1.

**Comment by NicolasDepoilly** [comment]
1 month and no answer on this, last commit 22 days ago is this project still in progress?

**Comment by lugovsky** [comment]
@NicolasDepoilly yes it's in progress, I believe it should work with newer versions. There's just other important stuff at the moment that we need to take care of.

In any case, updating dependency is kinda of a big deal, because we need to carefully test everything.

**Comment by NicolasDepoilly** [comment]
@lugovsky good to hear!
to be honest I am starting a new project from scratch with ng-admin and i used some ng2-smart-table. I was completely lost and thought I had to change of idea until I saw it was just a dependency issue.

Do your best guys and thanks for the answer!

**Comment by lugovsky** [comment]
@NicolasDepoilly No worries :)

**Comment by siddharthaReddy** [comment]
I had the same warning of peer dependency while running "npm install". I ignored it, but when I tried to start the app with "npm start", it rejected saying "Ng2SmartTableModule is not a NgModule".

Any idea why its causing this error? did I install it wrong ?

**Comment by boris-ilijic** [comment]
@siddharthaReddy Probably you are using angular-cli. If so, see the issue linked below. There is also described workaround for that temporary problem.
https://github.com/angular/angular-cli/issues/3426
